go.mod

```diff
@@ -257,3 +257,5 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/livepeer/ai-worker => ./ai-worker
```

Comment:
Seems like we have to update the ai-worker dependency still using `go get github.com/livepeer/ai-worker`